src/Symfony/Component/Console/Attribute/AsCommand.php

```diff
@@ -21,7 +21,7 @@ public function __construct(
         public string $name,
         public ?string $description = null,
         array $aliases = [],
-        bool $hidden = false,
+        public bool $hidden = false,
```

Comment:
Is this considered a BC break?

Reply:
Adding a public property to a class is allowed. 

https://symfony.com/doc/current/contributing/code/bc.html#changing-classes